README.md

```diff
@@ -238,4 +238,4 @@ fn main() {
 
 # Other Exchanges
 
-If you use [Bitfinex](https://www.bitfinex.com/) check out my [Rust library for bitfinex API](https://github.com/wisespace-io/bitfinex-rs)
\ No newline at end of file
+If you use [Bitfinex](https://www.bitfinex.com/) check out my [Rust library for bitfinex API](https://github.com/wisespace-io/bitfinex-rs)
```

Comment:
Not sure why GitHub highlights this one ... there was no newline change ... null terminator diff maybe?

Reply:
Yes, I think so. Thank you